### PR TITLE
feat(plugin): Add shellcheck plugin.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### What does this PR do?

Add shellcheck plugin to lint shell scripts. Useful tool for dog fooding. Ci che-plugin-registry and che-devfile-registry uses the same tool under the hood to validate shell scripts changes.

### CODE REVIEW

File: v3/plugins/timonwong/shellcheck/0.9.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+displayName: ShellCheck
+title: shellcheck
+description: 
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/timonwong/vscode-shellcheck
+category: Language
+firstPublicationDate: '2020-08-21'
+spec:
+  containers:
+  - image: "quay.io/aandriienko/che-sidecar-shellcheck:v0.7.1-d3cfda6"
```
[the remark this answers is not in the document]
Comment (benoitf) [reply v3/plugins/timonwong/shellcheck/0.9.0/meta.yaml]: I fixed the image

**Comment by benoitf** [comment]
I've updated this PR by bumping the version and fixing current issues

I've created a test devfile at https://gist.githubusercontent.com/benoitf/ada740277cd18769fcb6b05910683742/raw/f671aad0d9cc24f5f2b2c9cff8355c360952ebed/devfile-shellcheck.yaml

it requires to set `shellcheck.executablePath: "/bin/shellcheck"`

so to test 
https://che.openshift.io/f?url=https://gist.githubusercontent.com/benoitf/ada740277cd18769fcb6b05910683742/raw/f671aad0d9cc24f5f2b2c9cff8355c360952ebed/devfile-shellcheck.yaml

image

**Review by benoitf — APPROVED** [review]

**Comment by AndrienkoAleksandr** [comment]
@benoitf We have bash debugger, bash lsp and after this pr bash linter. Maybe we should create devfile for bash and include it to the stack list Eclipse Che? What Do you think?

**Comment by benoitf** [comment]
@AndrienkoAleksandr is it something you want to contribute ?

**Comment by AndrienkoAleksandr** [comment]
> @AndrienkoAleksandr is it something you want to contribute ?

I applied one minor commit. I think we've done. I hope that this minor commit won't affect ci.

**Comment by benoitf** [comment]
@ericwill ok for you ?

**Review by svor — APPROVED** [review]

**Review by ericwill — APPROVED** [review]

**Pull request merged**